gradio/blocks.py

```diff
@@ -1093,13 +1093,21 @@ def load(
         fn: Optional[Callable] = None,
         inputs: Optional[List[Component]] = None,
         outputs: Optional[List[Component]] = None,
+        api_name: AnyStr = None,
+        scroll_to_output: bool = False,
+        show_progress: bool = True,
+        queue=None,
+        batch: bool = False,
+        max_batch_size: int = 4,
+        preprocess: bool = True,
+        postprocess: bool = True,
+        every: float | None = None,
+        _js: Optional[str] = None,
         *,
         name: Optional[str] = None,
         src: Optional[str] = None,
         api_key: Optional[str] = None,
         alias: Optional[str] = None,
-        _js: Optional[str] = None,
-        every: None | int = None,
```

Comment:
No problem bringing a parameter after "*" to before "*" (the other way around would cause backwards incompatibility)

Reply:
Looks like gh markdown messed up what you're trying to say? I don't understand this

Reply:
Whoops I meant to say that it's safe to make the `_js` and `every` positional parameters instead of just named parameters. Wouldn't have been safe the other way around